include/etl/priority_queue.h

```diff
@@ -306,17 +306,27 @@ namespace etl
     }
 
     //*************************************************************************
-    /// Removes the oldest value from the back of the priority queue.
+    /// Removes the highest priority value from the top of the priority queue.
     /// Does nothing if the priority queue is already empty.
     //*************************************************************************
     void pop()
     {
-      // Move largest element to end
+      // Move largest (highest priority) element to end
       etl::pop_heap(container.begin(), container.end(), compare);
       // Actually remove largest element at end
       container.pop_back();
     }
 
+    //*************************************************************************
+    /// Removes the lowest priority value from the back of the priority queue.
+    /// Does nothing if the priority queue is already empty.
+    //*************************************************************************
+    void pop_back()
+    {
+      // Remove largest element at end
```

Comment:
The etl comments are a little confusing here since the size is just the default sorting algorithm. We are using `std::greater` and the default for the `etl::priority_queue` is `etl::less`. 

Should this be "Remove lowest priority element" instead?